packages/graphback-cli/src/helpers/db.ts

```diff
@@ -54,30 +33,35 @@ export const createDBResources = async (configInstance: ConfigBuilder, initializ
       await execa('touch', ['db.sqlite'])
     }
 
+    const dbConfig = {
+      client: config.db.database,
+      connection: config.db.dbConfig
+    }
+
     const schemaText = loadSchema(folders.model);
 
-    await migrate(schemaText, initializationStrategy);
+    const schemaWithDirectives = applyGeneratorDirectives(schemaText);
```

Comment:
[RequiresDiscussion] By implementing migration this way we assume that model files are valid schema (and we trying to make them valid). This is very strong design shift from the initial graphback and it will require some refactor in other layers. If we want to keep migration to be based on schema then we should probably pass the generated schema instead? The trick is that we might not know directives that were applied to the schema or scalars etc. 

Building schema this way - without providing resolvers/scalars might end up with validation errors. 
We need to find the best solution for this/document it and make sure that everything in ecosystem follows it.